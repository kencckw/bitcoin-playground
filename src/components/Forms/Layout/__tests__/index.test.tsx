import React from 'react';
import renderer from 'react-test-renderer';
import { FormLayout as Component } from '../';

it('renders correctly', () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
