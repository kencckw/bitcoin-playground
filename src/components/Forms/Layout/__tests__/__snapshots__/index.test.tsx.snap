// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-row ant-row-center ant-row-top"
  style={
    Object {
      "paddingTop": 12,
    }
  }
>
  <div
    className="ant-col ant-col-sm-24 ant-col-lg-16 ant-col-xl-12"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <div
      className="ant-card ant-card-bordered"
      style={
        Object {
          "minHeight": 650,
        }
      }
    >
      <div
        className="ant-card-body"
        style={Object {}}
      />
    </div>
  </div>
</div>
`;
